packages/effects-runtime/examples/src/passingDataTest.js

```diff
@@ -1,19 +1,19 @@
-require('../../lib/prelude-polyfill');
+require("../../lib/prelude-polyfill");
 
 const timesTwoHandler = (number) => {
-    recall (number * 2)
+  recall (number * 2);
 };
 
 const main = () => {
-    const result = perform ({type : 'timesTwo', number : 5});
-    console.log(result);
+  const result = perform { type: "timesTwo", number: 5 };
+  console.log(result);
 };
 
-'use effects';
-try{
-    main();
-} handle default with (e){
-    if(e.type === 'timesTwo'){
-        timesTwoHandler(e.number);
-    }
+("use effects");
+try {
+  main();
+} handle default  with (e) {
```

Comment:
wtf nooo! ok :face_with_head_bandage: i took a shortcut, will figure out how to do it right